oms-scomacs-solution/azuredeploy.json

```diff
@@ -0,0 +1,1504 @@
+                                    }
+                                }
+                            }
+                        }
+                    }
+                },
+                {
+                    "apiVersion": "[variables('LogAnalyticsApiVersion')]",
+                    "name": "ACSSavedSearch01",
+                    "type": "savedSearches",
+                    "dependsOn": [
+                        "[concat('Microsoft.OperationalInsights/workspaces/', parameters('workspaceName'))]"
+                    ],
+                    "properties": {
+                        "Category": "ACS Report Queries",
+                        "ETag": "*",
+                        "DisplayName": "Access Violation: Account Locked (Details)",
+                        "Query": "Type=SecurityEvent EventID=539 OR EventID=644 OR EventID=4740 OR EventID=6279",
+                        "Version": 1
+                    }
+                },
+                {
+                    "apiVersion": "[variables('LogAnalyticsApiVersion')]",
+                    "name": "ACSSavedSearch02",
+                    "type": "savedSearches",
+                    "dependsOn": [
+                        "[concat('Microsoft.OperationalInsights/workspaces/', parameters('workspaceName'))]"
+                    ],
+                    "properties": {
+                        "Category": "ACS Report Queries",
+                        "ETag": "*",
+                        "DisplayName": "Access Violation: Account Locked (Events Count)",
+                        "Query": "Type=SecurityEvent EventID=539 OR EventID=644 OR EventID=4740 OR EventID=6279 | measure count() by EventID",
+                        "Version": 1
+                    }
+                },
+                {
+                    "apiVersion": "[variables('LogAnalyticsApiVersion')]",
+                    "name": "ACSSavedSearch03",
+                    "type": "savedSearches",
+                    "dependsOn": [
+                        "[concat('Microsoft.OperationalInsights/workspaces/', parameters('workspaceName'))]"
+                    ],
+                    "properties": {
+                        "Category": "ACS Report Queries",
+                        "ETag": "*",
+                        "DisplayName": "Access Violation: Unsuccessful Logon Attempts (Details)",
+                        "Query": "Type=SecurityEvent EventID:[529..537] OR EventID=539 OR (EventID=4625 AND Status=0xc000006d) | Select TargetAccount, IpAddress, Computer, LogonProcessName, AuthenticationPackageName, LogonTypeName",
+                        "Version": 1
+                    }
+                },
+                {
+                    "apiVersion": "[variables('LogAnalyticsApiVersion')]",
+                    "name": "ACSSavedSearch04",
+                    "type": "savedSearches",
+                    "dependsOn": [
+                        "[concat('Microsoft.OperationalInsig
```

Comment:
type the apiVersion on the resources directly, instead of using the apiVersion in a variable. This is for best practices and tooling